src/features/presentation/layouts/presentation/presentation.layout.html

```diff
@@ -0,0 +1,141 @@
+      Intégrer la cartographie
+    </button>
+  </div>
+</div>
+<div class="bg-presentation-bg d-flex flex-column align-items-center justify-content-center">
+  <div class="container mb-5" style="width: 60%">
+    <h1 class="h3 fw-bold my-5 text-black">Une nouvelle cartographie ? Pourquoi ?</h1>
+    <p>
+      L’objectif de cette cartographie est de participer à l’amélioration de l'accompagnement proposé par les professionnels de
+      la médiation numérique.
+      <br />
+    </p>
+    <p>
+      L'idéologie de celle-ci est de proposer un outil à la fois national et local. La base de données est open-source et
+      contributive, de la plus petite collectivité à un hub, peut implémenter sur son site cet outil et afficher l'offre de
+      médiation de son territoire. Travailler sur cette base de donnes commune permet d'avoir une gestion de la donnée
+      centralisée, davantage actualisée et permettra d'améliorer la visibilité et le travail de ces structures de médiation
+      numérique.
+    </p>
+  </div>
+</div>
+<div class="container" *ngIf="!this.environment">
+  <h1 class="h3 fw-bold my-5 text-black text-center">De quel écosystème parle-t-on ?</h1>
+  <div class="row d-flex justify-content-center">
+    <div class="col-auto">
+      <div class="card" style="width: 18rem; border-radius: 0">
+        <img src="../../../../../../assets/img/presentation/svg-dora.svg" class="img-fluid" alt="image_dora" />
+        <div class="card-body">
+          <h1 class="h5 fw-bold card-title">Dora</h1>
+          <p class="card-text">
+            Vous travaillez dans une structure de médiation numérique ? Utilisez les formulaires Dora afin de renseigner vos
+            données !
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="col-auto">
+      <div class="card" style="width: 18rem; border-radius: 0">
+        <img src="../../../../../../assets/img/presentation/svg-rdv.svg" class="img-fluid" alt="image_rdv" />
+        <div class="card-body" style="min-height: 10rem">
+          <h1 class="h5 fw-bold card-title">RDV Solidarité</h1>
+          <p class="card-text">La prise de Rendez-vous est désormais possible directement depuis la cartographie.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row d-flex justify-content-center my-5">
+    <div class="col-auto">
+      <div class="card" style="width: 18rem; border-radius: 0">
+        <img src="../../../../../../assets/img/presentation/svg-labase.svg" class="img-fluid" alt="image_labase" />
+        <div class="card-body">
+          <h1 class="h5 fw-bold card-title">La Base</h1>
+          <p class="card-text">
+            La Base regroupe les ressources des professionnels du numérique d’intérêt général dont l’inclusion et la médiation
+            numériques.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="col-auto">
+      <div class="card" style="width: 18rem; border-radius: 0">
```

Comment:
```suggestion
      <div class="card rounded-0">
```
À quoi sert style="width: 18rem" ici ?